library/mbedtls.pc.in

```diff
@@ -0,0 +1,11 @@
+prefix="@PKGCONFIG_PREFIX@"
+libdir=@PKGCONFIG_LIBDIR@
+includedir=@PKGCONFIG_INCLUDEDIR@
+
+Name: mbedtls
+Description: The mbedTLS TLS library
+Version: @MBEDTLS_VERSION_STRING@
+Libs: -L${libdir} -lmbedtls
+Libs.private: @PKGCONFIG_LIBS@
+Requires.private: @PKGCONFIG_REQUIRES@
```

Comment:
Add mbedx509 and mbedcrypto to Requires.private as they are required by libmbedtls